client/scripts/tests-shim.js

```diff
@@ -1,3 +1,12 @@
 const { GlobalRegistrator } = require("@happy-dom/global-registrator")
 
 GlobalRegistrator.register()
+
+// this is a filthy hack
```

Comment:
i'd like to point out that it hurt my soul to write this hack

it works though lmao. `happy-dom` apparently doesn't have `matchMedia` implemented, probably because that would require implementing a ton of the CSS spec